packages/kbn-text-based-editor/src/text_based_languages_editor.tsx

```diff
@@ -83,9 +83,17 @@ export const TextBasedLanguagesEditor = memo(function TextBasedLanguagesEditor({
   const datePickerOpenStatusRef = useRef<boolean>(false);
   const { euiTheme } = useEuiTheme();
   const kibana = useKibana<TextBasedEditorDeps>();
-  const { dataViews, expressions, indexManagementApiService, application, core, fieldsMetadata } =
-    kibana.services;
+  const {
+    dataViews,
+    expressions,
+    indexManagementApiService,
+    application,
+    core,
+    fieldsMetadata,
+    uiSettings,
+  } = kibana.services;
   const timeZone = core?.uiSettings?.get('dateFormat:tz');
+  const histogramBarTarget = uiSettings?.get('histogram:barTarget');
```

Comment:
@qn895 here change to  

```
const histogramBarTarget = uiSettings?.get('histogram:barTarget') ?? 50;
```

in case this doesnt exist. Tbh I dont think that any user is changing this default value.

Reply:
The [getAddDateHistogramSnippet()](https://github.com/elastic/kibana/pull/192038/files#diff-5cd9c8ee459a745d21335b5264f9181876a28f7f66ab9582206fcc9fa042cc98R34) is already defaulting to 50 if `uiSettings?.get('histogram:barTarget')` is undefined. But I added the fallback for extra clarity here [`fdb2ad1` (#192038)](https://github.com/elastic/kibana/pull/192038/commits/fdb2ad1dc51b7ff191e5a9d4a6a8935f4044cbaf)

Reply:
Awesome